### PR TITLE
allow to list only certain resource types

### PR DESCRIPTION
**Pull request by thirdeyenick**
This allows to only show certain resource types when listing all existing resources. It helps in finding resources without the need for a specific nctl implementation.

### CODE REVIEW

File: get/all.go [hunk 1]
```diff
@@ -134,6 +145,26 @@ func printItems(items []*unstructured.Unstructured, get Cmd, out io.Writer, head
...
+		return lists, nil
+	}
+OUTER:
+	for _, kind := range kinds {
+		for _, list := range lists {
+			if !strings.EqualFold(kind+"list", list.GroupKind().Kind) {
+				continue
+			}
+			result = append(result, list)
+			continue OUTER
```
Comment (ctrox) [inline get/all.go:161]: isn't it simpler to just `break` here?

Comment (thirdeyenick) [reply get/all.go:161]: yeah, you are right 🤦

Comment (thirdeyenick) [reply get/all.go:161]: It actually doesn't work as the error gets thrown when we found a list type for a requested kind now. So we can either leave it with the `label` or we introduce a `bool` var which allows to find out if something was found. Wdyt?

File: get/all.go [hunk 2]
```diff
@@ -70,11 +71,21 @@ func projectNames(projects []management.Project) []string {
 	return result
 }
 
-func getProjectContent(ctx context.Context, client *api.Client, projNames []string, includeNineOwned bool) ([]*unstructured.Unstructured, []string, error) {
+func getProjectContent(
```
[the remark this answers is not in the document]
Comment (thirdeyenick) [reply get/all.go]: Yeah good idea. Fixed it.

File: get/all.go (hunk 1 above)
Comment (ctrox) [reply get/all.go:155]: Ah yeah makes sense. Both is fine with me honestly.

**Review by ctrox — APPROVED** [review]

File: get/all.go (hunk 1 above)
Comment (thirdeyenick) [reply get/all.go:155]: I'll leave it then as it is. 👍

**Pull request merged**